### ISSUE TITLE
How to use mtcnn tflite models (3 tflite files) ?

### ISSUE THREAD

**Issue by oopsodd** [jiangxiluning/facenet_mtcnn_to_mobile#3]
Does anyone has the python example to inference mtcnn tflite?

**Comment by jiangxiluning** [comment]
I think the official docs could tell u how to do it.

**Comment by chenyuqing** [comment]
No, I think the official docs are too hard to read. cry cry cry... So is there any example code here ? Thanks!

**Comment by Rob1datta** [comment]
I am trying to put an example code together for mtcnn tflite(3 models) by referring to object detection tflite code i had written earlier.But its not helping for mtcnn model Any leads or sample code will be appreciable.Thank you.

**Comment by ghost** [comment]
Any help on inference of mtcnn tflite models using python ?

**Comment by anneyking** [comment]
@oopsodd 
你好，请问你的facenet.tflite转换成功了吗?可以发一份给我吗？
